internal/docs/readme.go

```diff
@@ -47,66 +70,90 @@ func IsReadmeUpToDate() (bool, error) {
+		}
+
+		targets = append(targets, target)
+	}
+	return targets, nil
+}
+
+func updateReadme(filename, packageRoot string) (string, error) {
+	logger.Debugf("Update the %s file", filename)
 
 	packageRoot, err := packages.MustFindPackageRoot()
 	if err != nil {
 		return "", errors.Wrap(err, "package root not found")
 	}
 
-	rendered, shouldBeRendered, err := generateReadme(packageRoot)
+	rendered, shouldBeRendered, err := generateReadme(filename, packageRoot)
 	if err != nil {
 		return "", err
 	}
 	if !shouldBeRendered {
 		return "", nil
 	}
 
-	target, err := writeReadme(packageRoot, rendered)
+	target, err := writeReadme(filename, packageRoot, rendered)
 	if err != nil {
-		return "", errors.Wrapf(err, "writing %s file failed", ReadmeFile)
+		return "", errors.Wrapf(err, "writing %s file failed", filename)
 	}
 	return target, nil
 }
 
-func generateReadme(packageRoot string) ([]byte, bool, error) {
-	logger.Debugf("Generate %s file (package: %s)", ReadmeFile, packageRoot)
-	templatePath, found, err := findReadmeTemplatePath(packageRoot)
+func generateReadme(fileName, packageRoot string) ([]byte, bool, error) {
+	logger.Debugf("Generate %s file (package: %s)", fileName, packageRoot)
+	templatePath, found, err := findReadmeTemplatePath(fileName, packageRoot)
 	if err != nil {
-		return nil, false, errors.Wrapf(err, "can't locate %s template file", ReadmeFile)
+		return nil, false, errors.Wrapf(err, "can't locate %s template file", fileName)
 	}
 	if !found {
 		logger.Debug("README file is static, can't be generated from the template file")
 		return nil, false, nil
 	}
-	logger.Debugf("Template file for %s found: %s", ReadmeFile, templatePath)
+	logger.Debugf("Template file for %s found: %s", fileName, templatePath)
 
-	rendered, err := renderReadme(packageRoot, templatePath)
+	rendered, err := renderReadme(fileName, packageRoot, templatePath)
 	if err != nil {
 		return nil, true, errors.Wrap(err, "rendering Readme failed")
 	}
 	return rendered, true, nil
 }
 
-func findReadmeTemplatePath(packageRoot string) (string, bool, error) {
-	templatePath := filepath.Join(packageRoot, "_dev", "build", "docs", ReadmeFile)
+func findReadmeTemplatePath(fileName, packageRoot string) (string, bool, error) {
+	templatePath := filepath.Join(packageRoot, "_dev", "build", "docs", fileName)
 	_, err := os.Stat(templatePath)
 	if err != nil && os.IsNotExist(err) {
 		return "", false, nil // README.md file not found
 	}
 	if err != nil {
-		return "", false, errors.Wrapf(err, "can't located the %s file", ReadmeFile)
+		return "", false, errors.Wrapf(err, "can't located the %s file", fileName)
 	}
 	return templatePath, true, nil
 }
 
-func renderReadme(packageRoot, templatePath string) ([]byte, error) {
-	logger.Debugf("Render %s file (package: %s, templatePath: %s)", ReadmeFile, packageRoot, templatePath)
+func renderReadme(fileName, packageRoot, templatePath string) ([]byte, error) {
```

Comment:
nit: naming consistency. In this PR you're using `fileName` and `filename`. It would be nice to use either upper or lower capital letters.

Reply:
Done! all changed to `fileName`.